src/frontend/mosh-server.cc

```diff
@@ -502,6 +517,14 @@ static int run_server( const char *desired_ip, const char *desired_port,
       exit( 1 );
     }
 
+    /* set SSH_AUTH_SOCK */
+    if ( agent.active() ) {
+      if ( setenv( "SSH_AUTH_SOCK", agent.listener_path().c_str(), true ) < 0 ) {
+	perror( "setenv" );
```

Comment:
Well, since SSH_AUTH_SOCK is well known environment variable and can't be changed because it's what ssh uses, there isn't much point in defining it. getenv("SSH_AUTH_SOCK") i.m.o. is easier to write and understand than getenv(SSH_AUTH_SOCK). All the latter provides with, is a possibility to break the code in a subtle way by changing the value of the defined constant. It's almost as useful as:

```
#define HOME_DIRECTORY_ENV_NAME "HOME"
...
getenv(HOME_DIRACTORY_ENV_NAME)
...
```